src/aiidalab_qe/plugins/xps/result.py

```diff
@@ -0,0 +1,247 @@
+                points = chemical_shifts
+                xaxis = "Chemical Shift (eV)"
+            else:
+                points = binding_energies
+                xaxis = "Binding Energy (eV)"
+            #
+            spectra = xps_spectra_broadening(
+                points, equivalent_sites_data, gamma=gamma.value, sigma=sigma.value
+            )
+
+            for site, d in spectra[spectrum_select.value].items():
+                data.append(
+                    {
+                        "x": d[0],
+                        "y": d[1],
+                        "site": site,
+                    }
+                )
+
+            with g.batch_update():
+                if len(g.data) == len(data):
+                    for i in range(len(data)):
+                        g.data[i].x = data[i]["x"]
+                        g.data[i].y = data[i]["y"]
+                        if fill.value:
+                            g.data[i].fill = "tozeroy"
+                        else:
+                            g.data[i].fill = None
+                        g.data[i].name = data[i]["site"].replace("_", " ")
+
+                else:
+                    g.data = []
+                    for d in data:
+                        g.add_scatter(
+                            x=d["x"], y=d["y"], fill=fill.value, name=d["site"]
+                        )
+                g.layout.barmode = "overlay"
+                g.layout.xaxis.title = xaxis
+
+        spectra_type.observe(response, names="value")
+        spectrum_select.observe(response, names="value")
+        gamma.observe(response, names="value")
+        sigma.observe(response, names="value")
+        fill.observe(response, names="value")
+        correction_energies_table = self._get_corrections()
+        self.children = [
+            spectra_type,
+            ipw.HBox(
+                children=[
+                    spectrum_select_prompt,
+                    spectrum_select,
+                ]
+            ),
+            voigt_profile_help,
+            paras,
+            fill,
+            g,
+            correction_energies_table,
+        ]
+
+    def _get_corrections(self):
+        from aiida.orm.utils.serialize import deserialize_unsafe
+
+        ui_parameters = self.node.base.extras.get("ui_parameters", {})
+        ui_parameters = deserialize_unsafe(ui_parameters)
+        correction_energies = ui_parameters["xps"]["correction_energies"]
+        # create a table for the correction energies using ipywidgets
+        correction_energies_table = ipw.HTML(
+            """<h4>Offset Energies (δ)</h4>
+            <div>The binding energies should be corrected by these constant offsets. These offsets mainly depend on chemical element and core level, and are determined by comparing the calculated core electron binding energy to the experimental one.</div>
+            <table><tr><th>Core-level</th><th>Value (eV)</th></tr>"""
+        )
```

Comment:
The first sentence should remind the user how the correction energies are used in practice:
```suggestion
        correction_energies_table = ipw.HTML(
            """<h4>Offset Energies (δ)</h4>
            <div>When comparing binding energies reported by AiiDALab-QE to experimental data, these should be corrected by the given offset listed below</div>
            <table><tr><th>Core-level</th><th>Value (eV)</th></tr>"""
        )
```

I would also recommend moving the second sentence (or maybe the relevant concept or description) to the documentation since this is not really needed here.

Reply:
thanks! I made the changes.